### DISCUSSION TITLE
Show name of actualy switched profile

### DISCUSSION THREAD

**Discussion by michalsiman** [foldynl/QLog#281] [Ideas]
It is possible show name of actualy switched profile somewhere on basic window? It is on card MY STATION, but i would like have it somewhere without switch to another sub-window. Maybe on TOP panel of window, where is "QLog" now, can be there a name of actual profile? I am changing profile lots of (POTA, WWFF, portable) and quickly check if i am on correct profile is important. Just idea. Thx

**Comment by foldynl** [comment]
I will try to improve it. In the left bottom, I would indicate not only the callsign, locator and name, but also the profile name from which it is derived.

issue #282 is opened for it
